google/ggrc-core
# GGRC-5095,4651 Issue tracker switcher is displayed for Assessment if tracker is turned off on Audit

# Issue description

If user turned off issue tracker for audit TICKET TRACKER is turned on for Assessment and Assessment template

# Steps to test the changes

1. Open any audit page and turn on issue tracker for an audit
2. Create assessment and assessment template with turned on issue tracker
3. Turn off issue tracker for audit
4. Go to other attributes tab on assessment and look at the ticket tracker info: TICKET TRACKER is turned on
5. Look at the assessment template info panel: TICKET TRACKER is turned on

_Actual Result_: If user turned off issue tracker for audit TICKET TRACKER is turned on for Assessment and Assessment template
_Expected Result_: If user turned off issue tracker for audit TICKET TRACKER should be turned ff for Assessment and Assessment template

# Solution description

Check only audit's issue tracker (`audit.issue_tracker.enabled`) property to display issue tracker section on Assessment/Assessment Template pages

# Sanity checklist




# PR Review checklist
